vimrc.bundles

```diff
@@ -8,11 +8,12 @@ else
   call plug#begin('~/.vim/plugged')
 endif
 
+Plug 'google/vim-maktaba'
+Plug 'google/vim-codefmt'
```

Comment:
This looks interesting and potentially useful for other formatters, such as:

```
  autocmd FileType java AutoFormatBuffer google-java-format
```

https://github.com/google/vim-codefmt#autoformatting

Reply:
One step at a time. :-)